README.md

```diff
@@ -1,3 +1,75 @@
 # ODH
 
 This repo contains manifests to install opendatahub.io
+
+## Folders
+
+This repository serves multiple purposes. To differentiate, we organize this repo in folders - each folder in the repo root per purpose
+
+```
+odh
+├── components
+│   └── data-catalog
+│       └── ...
+├── kfdef
+│   ├── jupyterhub
+│   │   ├── kfdef.yaml
+│   │   ├── kustomization.yaml
+│   │   └── overrides
+│   │       └── ...
+│   └── superset
+│       ├── kfdef.yaml
+│       ├── kustomization.yaml
+│       └── overrides
+│           └── ...
+└── operator
+    └── base
+        ├── ...
+        └── kustomization.yaml
+```
+
+### Operator
+
+Folder `operator` hosts manifests for ODH operator deployment. When deployed, it creates a subscription to this operator.
+
+### Kfdef
+
+`kfdef` folder hosts `kfdef.yaml` definitions for our deployments. For easier maintenance we prefer to deploy ODH components into separate namespaces. Each namespace has it's own sub-folder in the `kfdef`.
```

Comment:
> Each namespace has it's own sub-folder in the kfdef. 

The wording is a bit confusing here, maybe mention that each override/kfdef component folder should correspond to an individual namespace deployment, atm it seems to imply you have namespaces named "jupyterhub" "superset" etc.

Reply:
What about:

```
Each sub-folder in the `kfdef` folder hosts a deployment configuration for a single component or a closely related subset of components. When deployed, each of these configurations (sub-folders) can live in a separate namespace.
```

Is that better?

Reply:
yes much better!